### DISCUSSION TITLE
recipes/cookies-and-bars/buttery-pineapple-crumble-bars/

### DISCUSSION THREAD

**Discussion by giscus** [nicholaswilde/recipes#1071] [Comments]
# recipes/cookies-and-bars/buttery-pineapple-crumble-bars/

A collection of my recipes.

https://nicholaswilde.io/recipes/cookies-and-bars/buttery-pineapple-crumble-bars/

**Comment by nicholaswilde** [comment]
Used 1 layer of foil to line 8x8 glass pan. Leaked through and stuck to pan. Try double layer of foil and grease pan itself. Let cool in pan for significant amount of time before removing.
